manuals/2.0/ja/setup.md

```diff
@@ -2,7 +2,7 @@
 layout: docs2-ja
 title: Setting up your virtual host
 permalink: /manuals/2.0/ja/setup/
-previous_page: Models
+previous_page: Command line / console
```

Comment:
@kenjis , what I was asking you was translating the title and previous_page . 

Thanks!